### PR TITLE
Bug 1116128 - Adding some fuzz to the vertical swipe test. +autoland

### PR DESCRIPTION
**Pull request by etiennesegonzac**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[etiennesegonzac Etienne Segonzac (etiennesegonzac)](https://github.com/etiennesegonzac) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2a1e7ca253b2)

**Comment by try-server-hook** [comment]
[etiennesegonzac Etienne Segonzac (etiennesegonzac)](https://github.com/etiennesegonzac) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=fb009dfe884b)

**Comment by try-server-hook** [comment]
[etiennesegonzac Etienne Segonzac (etiennesegonzac)](https://github.com/etiennesegonzac) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0972ec3daeca)

**Comment by try-server-hook** [comment]
[etiennesegonzac Etienne Segonzac (etiennesegonzac)](https://github.com/etiennesegonzac) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8f6dd69e4918)

File: apps/system/test/marionette/edges_gesture_test.js [hunk 1]
```diff
@@ -111,15 +111,20 @@ marionette('Edges gesture >', function() {
   });
 
   // Bug 1116128 - Intermittent | Edges gesture > Swiping vertically
```
Comment (KevinGrandon) [inline apps/system/test/marionette/edges_gesture_test.js]: Can you remove this comment?

**Pull request closed without merging**